blocks/library/freeform/old-editor.js

```diff
@@ -3,6 +3,25 @@
  */
 import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { keycodes } from '@wordpress/utils';
+
+const { BACKSPACE, DELETE } = keycodes;
+
+function isTmceEmpty( editor ) {
+	// When tinyMce is empty the content seems to be:
+	// <p><br data-mce-bogus="1"></p>
+	// avoid expensive checks for large documents
+	const body = editor.getBody();
+	if ( body.childNodes > 1 ) {
```

Comment:
Isn't `childNodes` a `NodeList`. In other words, shouldn't this be checking `childNodes.length`, not `childNodes` itself?

Reply:
Yes it should!

Reply:
PR to fix here: https://github.com/WordPress/gutenberg/pull/2532

Reply:
@EphoxJames Could editor.dom.isEmpty not be used?

Reply:
If I'm reading the source correctly I don't believe it would do what we want. If we called it passing in the paragraph tag of <p><br/></p> then it would return true but if we called it with the body of the editor then it would return false (effectively passing the div tag of <div><p><br/><p></div> ) because it would consider the p tag to be content.
It would also do a lot more work internally.